docs/en/reference/unitofwork.rst

```diff
@@ -134,6 +134,10 @@ optimize the performance of the Flush Operation:
   explicit strategies of notifying the UnitOfWork what objects/properties
   changed.
 
+.. warning::
```

Comment:
```suggestion
.. note::
```

Because `note` was used in other deprecation boxes.

Reply:
Not quite I would say. 3 of 10 total deprecations are with warning.

![image](https://user-images.githubusercontent.com/3340882/120059237-d483e200-c058-11eb-95cf-a55a23c4404a.png)
![image](https://user-images.githubusercontent.com/3340882/120059243-dd74b380-c058-11eb-87d1-8396e3aae4a4.png)
![image](https://user-images.githubusercontent.com/3340882/120059247-e6658500-c058-11eb-8341-366cdea4d725.png)

Reply:
I refactored all other deprecations to note, for sake of consistency.